### PR TITLE
added navbar

### PR DESCRIPTION
**Pull request by cjinc121**
added navbar for desktop
Screenshot (118)

### CODE REVIEW

**Review by JantuDeb — COMMENTED** [review]
Add react router to your project then replace   <a > tag with Link

**Review by JantuDeb — APPROVED** [review]

**Review by tanishk15sharma — APPROVED** [review]
use <Link></Link> , other things are good.

File: gentleman/src/App.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 import Mockman from "mockman-js";
+import { Navbar } from "./components/navbar/Navbar.jsx";
+import { Routes, Route } from "react-router-dom";
+
 function App() {
   console.log(process.env);
   return (
     <div className="App">
-      <Mockman />
+      {/* <Mockman /> */}
```
Comment (deepak29-git) [inline gentleman/src/App.js]: remove unnecessary comment

File: gentleman/src/App.js [hunk 2]
```diff
@@ -1,10 +1,17 @@
 import "./App.css";
 import Mockman from "mockman-js";
+import { Navbar } from "./components/navbar/Navbar.jsx";
+import { Routes, Route } from "react-router-dom";
+
 function App() {
   console.log(process.env);
```
Comment (deepak29-git) [inline gentleman/src/App.js]: don't forget to remove console log

**Review by deepak29-git — COMMENTED** [review]
everything looks good to me

File: gentleman/src/App.js (hunk 1 above)
Comment (cjinc121) [reply gentleman/src/App.js]: done

File: gentleman/src/App.js (hunk 2 above)
Comment (cjinc121) [reply gentleman/src/App.js]: done

**Pull request merged**